dealroom_firestore_connector/__init__.py

```diff
@@ -292,14 +314,14 @@ def set_history_doc_refs(
     # CREATE: If there are not available documents in history
     elif len(history_refs) == 0:
         # Add any default values to the payload
-        _payload = {
+        _parsed_payload  = {
             "dealroom_id": _NOT_IN_DEALROOM_ENTITY_ID,
             "final_url": "",
-            **payload,
+            **parsed_payload,
```

Comment:
would use _payload, which means that we assign to _payload itself + some default vars if needed